Update index.html

alexandercyu opened the pull request:
Added text to DACCS project description
- added mention of collaboration and the specific agencies and universities
- added more text from proposal to describe goals of the project
- removed mention of "scientific gateway"

2023-11-03
------------
- Reworked opening paragraph into Mission and Vision paragraphs
- Added Funder images
- Merged Affiliation and Expertise for Francis Zweirs and Dan McKenney

2023-11-06
------------
Resolves #7
Resolves #8
Resolves #9

2023-11-07
------------
Made changes from today's meeting
- Moved opening sentence of Mission paragraph to second paragraph
- Added space to:
   - Between DACCS Project title and Mission and Vision sections
   - Above Mission and Vision titles
   - to left and right of Mission and Vision text 
 - Changed Mission and Vision background to white

alexandercyu committed:
Update index.html

Added text to DACCS project description
- added mention of collaboration and the specific agencies or universities named
- added more text from proposal to describe goals of the project

alexandercyu committed:
Update index.html

removed Science Gateway

alexandercyu committed:
Merge branch 'main' into content-revisions

alexandercyu committed:
Update daccs-project-css.css

- added css for mission statement box
- changed "cover-text-box" class to "cover-text-bg"
- added class for funder image size

alexandercyu committed:
Add Funder images

alexandercyu committed:
Update index.html

- Reworked opening paragraph into Mission and Vision statements
- Merged Affiliation and Expertise for Francis Zwiers and Dan McKenny
- Added Funder images

mishaschwartz reviewed (changes requested):
Please move the funder image updates to a new PR

alexandercyu committed:
Removed logo images for another PR

Added requirements file

alexandercyu committed:
Create requirements.txt

alexandercyu commented:
Removed Funder logos for another PR

alexandercyu committed:
Update index.html

Removed Funders logo code

mishaschwartz commented:
Please move requirements.txt updates to a new PR

mishaschwartz commented:
CSS updates should be moved to #10 with the rest of the changes

alexandercyu committed:
Merged project_description and content blocks into one block

alexandercyu commented:
Moved funders css to #10

alexandercyu committed:
Delete requirements.txt

Removed requirements for another PR

alexandercyu committed:
Update daccs-project-css.css

Removed Funders CSS changes for #10

alexandercyu committed:
Update index.html

Added credit for background image of Yosemite National Park

mishaschwartz commented on static/css/daccs-project-css.css:
```diff
@@ -4,10 +4,20 @@ body{
+    background-color: rgba(53, 84, 113, 0.3);
+}
+
+.mission-text-box{
```
Why have a different background and border radius for just this box?

alexandercyu replied:
It's to highlight the Mission statement.  Kind of like this at Ouranos:

https://www.ouranos.ca/en/qui-nous-sommes/mission-vision-et-valeurs

mishaschwartz replied:
Personally I think it looks out of place, especially because most of the text box is empty. But let's ask the rest of the team what they think

alexandercyu replied:
Background of text made white as per meeting

mishaschwartz commented on templates/site/index.html:
```diff
@@ -2,27 +2,53 @@
+                <div class="col-5 d-block mx-auto mission-text-box">
+                    <h2>Mission</h2>
+                    <p>
+                        Canada's data infrastructure is inadequate for working with the rapidly growing volume of climate and
```
This first sentence belongs in the second paragraph. It states the problem that is being solved by the project, this is unrelated to _who_ are the organizations participating in the project

alexandercyu replied:
Moved sentence to second paragraph

mishaschwartz commented on templates/site/index.html:
```diff
@@ -2,27 +2,53 @@
+                        Institute of Montréal (CRIM), and Ouranos.
+                    </p>
+                    <p>
+                        The Data Analytics for Canadian Climate Services (DACCS) project will simplify access to climate
```
You already introduced the acronym in the paragraph above, you don't need to spell it out again. Just use "DACCS" here

alexandercyu replied:
Got rid of full use of DACCS

mishaschwartz commented on templates/site/index.html:
```diff
@@ -2,27 +2,53 @@
+                    <p>
+                        The aim of the DACCS project is to drastically simplify the time-consuming
+                        tasks required to extract relevant climate information from large data sets, allowing researchers to
+                        focus on solving scientific problems rather than on data management issues.
```
"on data management" ("issues" is not needed)

alexandercyu replied:
Removed word "issues"

mishaschwartz commented on templates/site/index.html:
```diff
@@ -2,27 +2,53 @@
-         </div>
-    </div>
+            <div class="row">
+                <div class="col-5 d-block mx-auto mission-text-box">
```
I think this looks better as two rows, not side by side columns. But that's just personal preference

alexandercyu replied:
I have no preference so, changed to two rows

mishaschwartz commented on templates/site/index.html:
```diff
@@ -2,27 +2,53 @@
+                        focus on solving scientific problems rather than on data management issues.
+                    </p>
+                    <p>
+                        DACCS is devloping a platform called Marble that will facilitate the conversion of raw climate data
```
developing

alexandercyu replied:
Fixed spelling

alexandercyu committed:
Merge branch 'main' into content-revisions

alexandercyu committed:
Merge branch 'main' into content-revisions

alexandercyu committed:
Update index.html

- Added space buffer to:
   -  left and right of Mission and Vision text
   - Above Mission and Vision titles
   - between DACCS Project title and Mission and Vision paragraphs

- Moved opening sentence of Mission text to second paragraph
- Removed long form of DACCS name from second paragraph

alexandercyu committed:
Update index.html

Changed Mission and Vision background to white

alexandercyu committed:
Update index.html

- Made Mission and Vision sections into two rows on top of each other instead of side by side
- Fixed spelling
- Removed unneeded words
